File: APWeb/src/assets/model/InventoryReturn.ts
export class InventoryReturn {
  ProductID: string = "";
  StoreID: number = 0;
  ReturnQuantity!: number | undefined;
  Location!: string;
  ProductName!: string;
  SupplierID!: number | undefined;
  ReturnID!: number | undefined;
  Tax: number = 0;
  UnitCost!: number | undefined;
}

File: APWebBackEnd/WebApplication2/Models/InventoryReturn.cs
using System;

namespace APWeb.Models
{
    public class InventoryReturn
    {

        public int StoreID { get; set; }
        public string ProductID { get; set; }
        public string Location { get; set; }
        public int Units { get; set; }
        public string InspectedBy { get; set; }
        public int ReturnQuantity { get; set; }
        public int SupplierID { get; set; }
        public int ReturnID { get; set; }
        public string ProductName { get; set; }
        public float UnitCost { get; set; }
        public int Tax { get; set; }
    }
}
